server/datastore/mysql/policies.go

```diff
@@ -641,11 +641,11 @@ func (ds *Datastore) ListMergedTeamPolicies(ctx context.Context, teamID uint, op
 		FROM policies p
 		LEFT JOIN users u ON p.author_id = u.id
 		LEFT JOIN policy_stats ps ON p.id = ps.policy_id
-		AND ps.inherited_team_id = COALESCE(p.team_id, 0)
+		AND ps.inherited_team_id = IF(p.team_id IS NULL, ?, 0)
```

Comment:
Please remind me, inherited_team_id is only used for global policies right? (To not count global results of hosts in team B when on a team A domain)

Reply:
`inhertited_team_id` is `0` if the policy doesn't have inherited counts, ie if you want to get global counts, or if it's a team policy.

if `inherited_team_id  != 0` then its showing team counts for a global policy.

(there should be no entries for a team policy where `inherited_team_id != 0`)